packages/did-iplds/package.json

```diff
@@ -34,7 +34,7 @@
     "lint:fix": "yarn lint --fix",
     "size": "size-limit",
     "start": "rollup -cw",
-    "test": "yarn docker:start && yarn test:jest; yarn docker:stop",
+    "test": "yarn docker:start && yarn test:jest && yarn docker:stop",
```

Comment:
`&&` does not run next command when the previous fails. `;` runs command in any case. So in your case if tests fails the Docker containers will not stop, while with `;` the containers will stop in any case